assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -151,6 +151,12 @@ che.docker.ip=NULL
 # This is unusual, but happens for example in Docker for Mac when containers are in a VM.
 che.docker.ip.external=NULL
 
+# If true, then uses the internal address and port of workspace Docker containers (i.e. within the Docker
+# Docker network) instead of the docker host address and the port provided by the Docker daemon. May be necessary if the
+# Che server is unable to contact workspace agents, but will prevent communication if the server and workspace
+# are not on the same network.
+che.docker.ip.use_internal_address=false
```

Comment:
I've changed it so that property refers to strategy name, and strategies are injected in MapBinder.